plugins/org.jboss.tools.openshift.ui/src/org/jboss/tools/openshift/internal/ui/comparators/ProjectViewerComparator.java

```diff
@@ -10,16 +10,27 @@
  ******************************************************************************/
 package org.jboss.tools.openshift.internal.ui.comparators;
 
+import org.eclipse.jface.viewers.ILabelProvider;
 import org.eclipse.jface.viewers.Viewer;
-import org.eclipse.jface.viewers.ViewerComparator;
+import org.eclipse.jface.viewers.ViewerSorter;
+import org.jboss.tools.openshift.internal.ui.treeitem.ObservableTreeItem;
 
 import com.openshift.restclient.model.IProject;
+import com.openshift.restclient.model.IResource;
 
-public class ProjectViewerComparator extends ViewerComparator {
+public class ProjectViewerComparator extends ViewerSorter {
```

Comment:
sooooo... ViewerSorter is deprecated in favor of ViewerComparator https://bugs.eclipse.org/bugs/show_bug.cgi?id=402464